### PR TITLE
EVG-7364 gracefully handle missing results.

### PR DESCRIPTION
**Pull request by jimoleary**
It is possible that thread level do not contain any results. When this happens on a change point, walk the previous point back to find the correct place to draw the line segment.

### CODE REVIEW

**Comment by tychoish** [comment]
can you move the ticket to EVG (tig is private...)

File: public/static/app/perf/trend_chart.js [hunk 1]
```diff
@@ -710,18 +710,25 @@ mciModule.factory('DrawPerfTrendChart', function (
           'stroke-width': '4',
         });
 
-      changePointSegments.select('line')
+        const changePointIndex = (index, level, series) => {
+          for (;index >= 0 && getValueFor(level)(series[index]) == null;index--) {}
```
Comment (john-m-liu) [inline public/static/app/perf/trend_chart.js]: Could you use _.findLastIndex here? Or at least do a while loop which breaks once you've found the index. This was pretty confusing to read

**Comment by jimoleary** [comment]
moved to EVG-7364

**Comment by jimoleary** [comment]
failing tests are in test cloud and test third party. So I can't see how they are related to these changes.

**Pull request merged**